acme/resource_acme_certificate_test.go

```diff
@@ -367,6 +367,38 @@ func TestAccACMECertificate_httpMemcache(t *testing.T) {
 	})
 }
 
+func TestAccACMECertificate_httpS3(t *testing.T) {
+	wantEnv := os.Environ()
+	s3_bucket := os.Getenv("TF_S3_BUCKET")
+	// If bucket not set, skip test
+	if s3_bucket == "" {
+		t.Skip()
```

Comment:
_Note for @vancluever_ add a warning here mentioning that the test will be skipped due to missing env var